### ISSUE TITLE
Which data split was the final result coming from

### ISSUE THREAD

**Issue by zitongzhan** [dulucas/UVO_Challenge#12]
Hello! Many thanks for this nice GitHub repo. I am wondering on which data split did you evaluate your model. Whether it was `UVO sparse test` or `UVO dense test`? In the paper I saw the statement of `Challenge final results on UVO-Sparse test dataset`. But the testing script in Github code loaded test annotation from the dense split. I guess dense split's test set was what the challenge targeted?
![image](https://user-images.githubusercontent.com/24406547/174845442-0f115bf0-ad8f-43f1-84f9-03c2a402ed75.png)

**Comment by dulucas** [comment]
Hi !
The results are copied from the leaderboard of the challenge(which is unavailable on eval.ai but you can find a copy [here](https://sites.google.com/view/unidentified-video-object/challenge-intro)). 

I'm sorry that I can't recall precisely which part I used for evaluation, because it has been a long time after the challenge. But I do recall that the GT of the test part has not been released(only the images and IDs are given), the participants should upload their prediction to the server to get the final results. 

As for the testing script in Github code, I remember that I finetuned and evaluated my model using the UVO-sparse and UVO-dense(even though there is an overlapp between my training and testing dataset...) to select the best model.

Hope it helps !

**Comment by zitongzhan** [comment]
Hi, 

I just noticed that the model weights is no longer available. My team just sent you an email and attached an upload link for your convenience. Would you like to share that with me? Thanks a lot!

**Comment by dulucas** [comment]
Hi, so sorry for the late response because I was very busy these days, and Yes, I saw your email! I will share you the model pretrained on COCO and finetuned on UVO-train and UVO-validation. Unfortunately, I do have the model that pretrained on COCO and finetuned on UVO-train anymore due to the breakage of my server... Will do this today!

**Comment by dulucas** [comment]
Hi, all, I just uploaded all the models to my google drive and updated the links in the README, you should be able to download them now. BUT, please do it quick as the models have occupied almost all my drive storage and might affect my daily usage(receiving emails etc), so I might need to remove them in near future...😂 sorry for the inconvenience
